Raven.Abstractions/Smuggler/SmugglerDatabaseApiBase.cs

```diff
@@ -128,17 +128,17 @@ protected bool IgnoreErrorsAndContinue
                         await ExportIndexes(exportOptions.From, jsonWriter);
                     }
                     jsonWriter.WriteEndArray();
-
-                    // used to synchronize max returned values for put/delete operations
-                    var maxEtags = Operations.FetchCurrentMaxEtags();
-
+                    
+                    var lastDocDeleteEtag = Operations.FetchLastDocDeleteEtag();
                     jsonWriter.WritePropertyName("Docs");
                     jsonWriter.WriteStartArray();
                     if (Options.OperateOnTypes.HasFlag(ItemType.Documents))
                     {
                         try
                         {
-                            result.LastDocsEtag = await ExportDocuments(exportOptions.From, jsonWriter, result.LastDocsEtag, maxEtags.LastDocsEtag);
+                            result.LastDocsEtag = await ExportDocuments(exportOptions.From, jsonWriter, 
```

Comment:
not passing maxEtags means that we will never finish exporting on high traffic systems